packages/mongodb-memory-server-core/src/util/MongoBinaryDownload.ts

```diff
@@ -75,15 +76,15 @@ export class MongoBinaryDownload {
     const binaryName = this.platform === 'win32' ? 'mongod.exe' : 'mongod';
     const mongodPath = path.resolve(this.downloadDir, this.version, binaryName);
 
-    if (await this.locationExists(mongodPath)) {
+    if (await pathExists(mongodPath)) {
       return mongodPath;
     }
 
     const mongoDBArchive = await this.startDownload();
     await this.extract(mongoDBArchive);
-    fs.unlinkSync(mongoDBArchive);
+    await promises.unlink(mongoDBArchive);
```

Comment:
Just a note: `promises` variable looks a little bit strange for me in raw code.
In our apps we are using the following style (if Node>v14): 
```
import fs from 'fs/promises';

await fs.unlink...
```
Or we are using such style for Node>v10 – `fs.promises.unlink` which is more clear for newcomers

But feel free to leave the current style (without `fs`) without changes.

Reply:
i could also rename it, like i have done in `utils.ts` to `fspromises`

or just re-export these from `utils.ts`

Reply:
fspromises also looks good

Reply:
fixed in f876670aaecc3b74ca4a85a80d804c137a40d572